Modules/POSIX/Sources/POSIX/FileDescriptor.swift

```diff
@@ -1 +1,31 @@
+#if os(Linux)
+    import Glibc
+#else
+    import Darwin.C
+#endif
+
 public typealias FileDescriptor = Int32
+
+public func statusflags(fileDescriptor: FileDescriptor) throws -> Int32 {
+    let flags = fcntl(fileDescriptor, F_GETFL, 0)
+
+    if flags == -1 {
```

Comment:
This should be defensive

`guard flags != -1` and throw the error in the else statement